ui/app/pages/send/send-content/send-to-row/send-to-row.utils.js

```diff
@@ -17,9 +17,8 @@ function getToErrorObject (to, toError = null, hasHexData = false, tokens = [],
     }
   } else if (!isValidAddress(to, network) && !toError) {
     toError = isEthNetwork(network) ? INVALID_RECIPIENT_ADDRESS_ERROR : INVALID_RECIPIENT_ADDRESS_NOT_ETH_NETWORK_ERROR
-  } else if (selectedToken && (ethUtil.toChecksumAddress(to) in contractMap || checkExistingAddresses(to, tokens))) {
-    toError = KNOWN_RECIPIENT_ADDRESS_ERROR
```

Comment:
Hey @estebanmino ! Would love your review here. The issues stems from `KNOWN_RECIPIENT_ADDRESS_ERROR` being counted as a `toError`, and as a result, blocking the send. However, this util method actually hadn't changed since you implemented it, and somehow this only surface now. 

Questions for you:
- was the inclusion of `KNOWN_RECIPIENT_ADDRESS_ERROR` in `toError` an expected behavior?
- do you know how `toWarning` is prioritized over `toError` previously? It seems that `KNOWN_RECIPIENT_ADDRESS_ERROR` was always added to `toError`, but it wasn't showing as error until now.
- do you see any problems with removing this from util?

Reply:
`was the inclusion of KNOWN_RECIPIENT_ADDRESS_ERROR in toError an expected behavior` : No, I'm checking my PR now and I added a new method for warnings  https://github.com/MetaMask/metamask-extension/pull/6058/files#diff-78b29f57cf9c6b8206cbce06ea7f28bfR25 ... these are suppose to be non blocker because is added to `toWarning`.
`do you know how toWarning is prioritized over toError previously? It seems that `: I added `toWarning` because of this, we didn't want to block the tx because of warnings, so there was no prioritization of it. Now we show `toError` first, if there is no error we show `toWarning`.
`do you see any problems with removing this from util`:  I don't think it would be a problem, I remember now we had this behavior merged first so this was an error, then we decided to change it so this shouldn't be here, it's already handled as a warning.